src/hotspot/share/gc/shenandoah/shenandoahStackWatermark.cpp

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2020, Red Hat, Inc. All rights reserved.
```

Comment:
This is structurally similar to zStackWatermark.cpp, and perhaps 'inspired' by it. Consider adding Oracle copyright there in addition to the Red Hat copyright. We did the same in e.g. shenandoahReferenceProcessor.hpp/cpp. Also, bump the copyright year to 2021 maybe?